web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -797,6 +797,7 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
     if (this.usesFixedWidthScaling) {
       let baseWidth = this.width;
       baseWidth -= this._borderWidth * 2;
+      baseWidth = (isNaN(baseWidth) || baseWidth < 0) ? 0 : baseWidth;
```

Comment:
Does get internalHeight() (l.820) have a similar pattern to check?

Reply:
Does a `baseWidth` of zero actually make sense?

Reply:
Good question. @jahorton Do you have any insights on that?